src/test/groovy/org/prebid/server/functional/model/request/auction/ActivityRule.groovy

```diff
@@ -0,0 +1,18 @@
+package org.prebid.server.functional.model.request.auction
+
+import groovy.transform.ToString
+
+@ToString(includeNames = true, ignoreNulls = true)
+class ActivityRule {
+
+    Condition condition
+    Boolean allow
+    List<String> privacyRegs
```

Comment:
For now `privacyRegs` not implemented, moreover it may change during phase 4. Better to not add it for now.